chart/openfaas/README.md

```diff
@@ -135,10 +135,14 @@ Additional OpenFaaS options in `values.yaml`.
 | `gateway.readTimeout` | Queue worker read timeout | `20s` |
 | `gateway.writeTimeout` | Queue worker write timeout | `20s` |
 | `gateway.upstreamTimeout` | Maximum duration of upstream function call | `20s` |
-| `gateway.scaleFromZero` | Enables an intercepting proxy which will scale any function from 0 replicas to the desired amount | `false` |
+| `gateway.scaleFromZero` | Enables an intercepting proxy which will scale any function from 0 replicas to the desired amount | `true` |
 | `queueWorker.ackWait` | Max duration of any async task/request | `30s` |
 | `openfaasImagePullPolicy` | Image pull policy for openfaas components, can change to `IfNotPresent` in offline env | `Always` |
 | `kubernetesDNSDomain` | Domain name of the Kubernetes cluster | `cluster.local` |
+| `faasIdler.inactivityDuration` | Duration after which faas-idler will scale function down to 0 | `5m` |
+| `faasIdler.reconcileInterval` | The time between each of reconciliation | `30s` |
+| `faasIdler.dryRun` | If set to true, faas-idler will just simulate scaling down functions (you can see how it works in logs) | `true` |
```

Comment:
When set to `false` the OpenFaaS API will be called to scale down idle functions, by default this is set to only print in the logs.